rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -999,7 +1000,7 @@ def try_insert_mapping(cur):
         oracles_after_upgrade = cursor.execute(
             'SELECT value FROM settings WHERE name="current_price_oracles"',
         ).fetchone()[0]
-        assert oracles_after_upgrade == '["manualcurrent", "cryptocompare", "coingecko", "defillama", "uniswapv2", "uniswapv3", "saddle"]'  # noqa: E501
+        assert oracles_after_upgrade == '["manualcurrent", "cryptocompare", "coingecko", "defillama", "uniswapv2", "uniswapv3"]'  # noqa: E501
```

Comment:
as I said in the post there is no need to remove saddle from upgrades before v38->v39